packages/goto/src/index.js

```diff
@@ -62,40 +29,11 @@ module.exports = async (
   }
 ) => {
   await page.setRequestInterception(true)
-  let reqCount = { abort: 0, continue: 0 }
-
-  page.on('request', req => {
-    const resourceUrl = req.url()
-    const resourceType = req.resourceType()
 
-    if (abortTypes.includes(resourceType)) {
-      debug(`abort:${resourceType}:${++reqCount.abort}`, resourceUrl)
-      return req.abort()
-    }
-
-    if (adblock) {
-      const frame = req.frame()
-
-      const { match: isMatch } = engine.match(
-        makeRequest(
-          {
-            type: webRequestType(resourceType),
-            sourceUrl: frame ? frame.url() : undefined,
-            url: resourceUrl
-          },
-          url => tldts.parse(url)
-        )
-      )
-
-      if (isMatch) {
-        debug(`abort:tracker:${++reqCount.abort}`, resourceUrl)
-        return req.abort()
-      }
-    }
-
-    debug(`continue:${resourceType}:${++reqCount.continue}`, resourceUrl)
-    return req.continue()
-  })
+  if (adblock) {
+    debug('enable adblocker')
```

Comment:
some `debug` logs were removed with this update.

is it a way to enable see what is doing `@cliqz/adblocker` behind the hood?

just a way to make easy debug traces

Reply:
Indeed, the debug logs for adblocker have been removed as there is currently no way to know which requests where blocked (everything happens internally in PuppeteerBlocker). If that is a desired capability, we could easily add a hook/callback to get some statistics about blocked requests. Currently I'm not sure what a good API for this would like like, but I'm open for suggestions.